scarlet/bbox.py

```diff
@@ -11,22 +11,17 @@ class Box:
     Parameters
     ----------
     shape: tuple
-        Size of the box in depth,height,width
+        Size of the box
     origin: tuple
-        Minimum (z,y,x) value of the box (front low left corner).
+        Front/low/left corner coordinate of the box
```

Comment:
This is still 3-dimensional. In the previous docstring there at least was a notion of minimum, which ignores the number of dimensions.